core/CMakeLists.txt

```diff
@@ -336,6 +336,10 @@ if (X86 AND X64)
   set(ARCH_SRCS ${ARCH_SRCS} arch/${ARCH_NAME}/x86_to_x64.c)
 endif (X86 AND X64)
 
+if (AARCHXX)
+  set(ARCH_SRCS ${ARCH_SRCS} arch/${ARCH_NAME}/offsets.c)
```

Comment:
I would agument the name: either `asm_offsets.{c,h}` or `struct_offsets.{c,h}`.

Reply:
Done.